client/my-sites/guided-transfer/host-credentials-page/index.jsx

```diff
@@ -29,16 +43,35 @@ class HostCredentialsPage extends Component {
 		this.setFieldValue( fieldName, e.target.value );
 	}
 
+	redirectToCart = () => {
```

Comment:
Not a blocker, but since this is only called in `componentWillReceiveProps`, we don't need to preserve the `this` context with the arrow function. Not really a blocker, and perhaps better to be on the safe side.